app/controllers/helpers/application_helper.rb

```diff
@@ -0,0 +1,2 @@
+module ApplicationHelper
+end
```

Comment:
What's this? I haven't come across this before. Maybe it's the file you'd been fighting with, but no idea what it does

Reply:
This was the file I had mentioned previously that I had to add from git